### ISSUE TITLE
Does Property Name Size Affect Tile Size?

### ISSUE THREAD

**Issue by jbeuckm** [mapbox/tippecanoe#885]
I have large numbers of point features and am hitting the 500k limit. Does the size of the property names affect the tilesize? So if I rename the property "longestKnownPropertyName" to "l" will this help me under the 500k limit?

**Comment by ericfischer** [comment]
Yes, the property name takes up as much space in the tile as a string value does. But it will not save very much if the same property name is used in each feature, because the name will only be stored once and then used by reference in each feature.
